scico/linop/radon_svmbir.py

```diff
@@ -190,20 +192,28 @@ class SVMBIRWeightedSquaredL2Loss(WeightedSquaredL2Loss):
     of :class:`scico.linop.Diagonal`. If :math:`W` is None, it is set to
     :class:`scico.linop.Identity`.
 
+    The extended loss differs from a typical weighted squared :math:`\ell_2` loss
+    is the following ways.
+
     When ``positivity=True``, the prox projects onto the non-negative
-    quadrant, but the the loss, :math:`\alpha l(\mb{y}, A(\mb{x}))`,
+    orthant, but the the loss, :math:`\alpha l(\mb{y}, A(\mb{x}))`,
     is unaffected by this setting and still evaluates to finite values
-    when :math:`\mb{x}` is not in the non-negative quadrant.
+    when :math:`\mb{x}` is not in the non-negative orthant.
```

Comment:
"nonnegative". Generally don't hyphenate prefixes (unless they are words themselves, or if needed for clarity, or...)

Reply:
Fixed